packages/web-worker/src/webpack-parts/loader.ts

```diff
@@ -0,0 +1,124 @@
+import * as path from 'path';
+
+import SingleEntryPlugin from 'webpack/lib/SingleEntryPlugin';
+import WebWorkerTemplatePlugin from 'webpack/lib/webworker/WebWorkerTemplatePlugin';
+import FetchCompileWasmTemplatePlugin from 'webpack/lib/web/FetchCompileWasmTemplatePlugin';
```

Comment:
This is interesting, what this wasm template for?

Reply:
This was from the https://github.com/GoogleChromeLabs/worker-plugin plugin, where I got a lot of the implementation from. It just allows you to import `.wasm` files IIRC, which is a good approach for things in workers.